<?xml version="1.0" encoding="utf-8"?>
<TestRun id="5da3c8ab-f1df-4cc1-8657-378cca962a9f" name="Jhonatan Medeiros@DESKTOP-8DRS4MO 2023-11-07 10:39:11" runUser="DESKTOP-8DRS4MO\Jhonatan Medeiros" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-11-07T10:39:11.6459672-03:00" queuing="2023-11-07T10:39:11.6459672-03:00" start="2023-11-07T10:39:11.2355166-03:00" finish="2023-11-07T10:39:11.6469678-03:00" />
  <TestSettings name="default" id="ed50e43d-5f15-409e-a9db-3f17ad62f534">
    <Deployment runDeploymentRoot="Jhonatan_Medeiros_DESKTOP-8DRS4MO_2023-11-07_10_39_11" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="d9504737-38ba-4273-8350-42c8c8d184d4" testId="928aa985-ca73-c448-5d8b-b6e6f1f3a744" testName="FileNameDoesExist" computerName="DESKTOP-8DRS4MO" duration="00:00:00.0067557" startTime="2023-11-07T10:39:11.5694620-03:00" endTime="2023-11-07T10:39:11.5979832-03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="d9504737-38ba-4273-8350-42c8c8d184d4">
      <Output>
        <StdOut>TestContext Messages:
In MyClassesTest.AssemblyInitCleanup.Assembly&#xD;
In FileProcessTest.ClassInitialize() method&#xD;
in FileProcessTest.TestInitialize() method&#xD;
 Test Purpose: Check to see if a file exists.
Test Owner: JhonatanM.&#xD;
Checking for File: 'C:\Users\Jhonatan Medeiros\AppData\Roaming\TestFile.txt'.</StdOut>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="3de833ab-8b6c-48b3-bcc0-1011e387307e" testId="6f90c9b0-a18c-f05c-3cc2-7cca266a69be" testName="FileNameDoesNotExist" computerName="DESKTOP-8DRS4MO" duration="00:00:00.0001723" startTime="2023-11-07T10:39:11.6029456-03:00" endTime="2023-11-07T10:39:11.6032182-03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="3de833ab-8b6c-48b3-bcc0-1011e387307e">
      <Output>
        <StdOut>TestContext Messages:
in FileProcessTest.TestInitialize() method&#xD;
 Test Purpose: Check to see if file does not exist.
Test Owner: JhonatanM.&#xD;
Checking for File: 'C:\NotExists.txt' does NOT exist.</StdOut>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="91db61bc-1971-4e82-8c32-701ae4b1f092" testId="8b48ffd8-8c07-5f7b-0eb6-d4aee87d6161" testName="SimulateTimeout" computerName="DESKTOP-8DRS4MO" startTime="2023-11-07T10:39:11.6047222-03:00" endTime="2023-11-07T10:39:11.6051863-03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="NotExecuted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="91db61bc-1971-4e82-8c32-701ae4b1f092">
      <Output></Output>
    </UnitTestResult>
    <UnitTestResult executionId="e7591804-9502-4017-99f9-73069e63f766" testId="0afa7d29-06f5-3a62-8ce9-439993ab8178" testName="FileNameNullOrEmpty_UsingExpectedExceptionAttribute" computerName="DESKTOP-8DRS4MO" duration="00:00:00.0011298" startTime="2023-11-07T10:39:11.6032837-03:00" endTime="2023-11-07T10:39:11.6044994-03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="e7591804-9502-4017-99f9-73069e63f766">
      <Output>
        <StdOut>TestContext Messages:
in FileProcessTest.TestInitialize() method&#xD;
 Test Purpose: Check for a thrown ArgumentNullException using ExpectedException.
Test Owner: JhonatanM.&#xD;
Checking for an empty file name.</StdOut>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="c0cb277d-555c-45e1-9da0-579173c4f4c4" testId="e40784d4-b509-7b24-dc06-ffcaafa43c7b" testName="FileNameNullOrEmpty_UsingTryCatch_ShouldReturnThrowArgumentNullException" computerName="DESKTOP-8DRS4MO" duration="00:00:00.0001296" startTime="2023-11-07T10:39:11.6045257-03:00" endTime="2023-11-07T10:39:11.6047033-03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="c0cb277d-555c-45e1-9da0-579173c4f4c4">
      <Output>
        <StdOut>TestContext Messages:
in FileProcessTest.TestInitialize() method&#xD;
 Test Purpose: Check for a thrown ArgumentNullException using ExpectedException.
Test Owner: JhonatanM.&#xD;
Checking for an empty file name.</StdOut>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="FileNameDoesExist" storage="c:\users\jhonatan medeiros\source\repos\myclasses\myclassestest\bin\debug\net6.0\myclassestest.dll" priority="3" id="928aa985-ca73-c448-5d8b-b6e6f1f3a744">
      <Owners>
        <Owner name="JhonatanM" />
      </Owners>
      <TestCategory>
        <TestCategoryItem TestCategory="NoException" />
      </TestCategory>
      <Execution id="d9504737-38ba-4273-8350-42c8c8d184d4" />
      <TestMethod codeBase="C:\Users\Jhonatan Medeiros\source\repos\MyClasses\MyClassesTest\bin\Debug\net6.0\MyClassesTest.dll" adapterTypeName="executor://mstestadapter/v2" className="MyClassesTest.FileProcessTest" name="FileNameDoesExist" />
    </UnitTest>
    <UnitTest name="SimulateTimeout" storage="c:\users\jhonatan medeiros\source\repos\myclasses\myclassestest\bin\debug\net6.0\myclassestest.dll" id="8b48ffd8-8c07-5f7b-0eb6-d4aee87d6161">
      <Execution id="91db61bc-1971-4e82-8c32-701ae4b1f092" />
      <TestMethod codeBase="C:\Users\Jhonatan Medeiros\source\repos\MyClasses\MyClassesTest\bin\Debug\net6.0\MyClassesTest.dll" adapterTypeName="executor://mstestadapter/v2" className="MyClassesTest.FileProcessTest" name="SimulateTimeout" />
    </UnitTest>
    <UnitTest name="FileNameNullOrEmpty_UsingExpectedExceptionAttribute" storage="c:\users\jhonatan medeiros\source\repos\myclasses\myclassestest\bin\debug\net6.0\myclassestest.dll" priority="1" id="0afa7d29-06f5-3a62-8ce9-439993ab8178">
      <Owners>
        <Owner name="JhonatanM" />
      </Owners>
      <TestCategory>
        <TestCategoryItem TestCategory="Exception" />
      </TestCategory>
      <Execution id="e7591804-9502-4017-99f9-73069e63f766" />
      <TestMethod codeBase="C:\Users\Jhonatan Medeiros\source\repos\MyClasses\MyClassesTest\bin\Debug\net6.0\MyClassesTest.dll" adapterTypeName="executor://mstestadapter/v2" className="MyClassesTest.FileProcessTest" name="FileNameNullOrEmpty_UsingExpectedExceptionAttribute" />
    </UnitTest>
    <UnitTest name="FileNameNullOrEmpty_UsingTryCatch_ShouldReturnThrowArgumentNullException" storage="c:\users\jhonatan medeiros\source\repos\myclasses\myclassestest\bin\debug\net6.0\myclassestest.dll" priority="2" id="e40784d4-b509-7b24-dc06-ffcaafa43c7b">
      <Owners>
        <Owner name="JhonatanM" />
      </Owners>
      <TestCategory>
        <TestCategoryItem TestCategory="Exception" />
      </TestCategory>
      <Execution id="c0cb277d-555c-45e1-9da0-579173c4f4c4" />
      <TestMethod codeBase="C:\Users\Jhonatan Medeiros\source\repos\MyClasses\MyClassesTest\bin\Debug\net6.0\MyClassesTest.dll" adapterTypeName="executor://mstestadapter/v2" className="MyClassesTest.FileProcessTest" name="FileNameNullOrEmpty_UsingTryCatch_ShouldReturnThrowArgumentNullException" />
    </UnitTest>
    <UnitTest name="FileNameDoesNotExist" storage="c:\users\jhonatan medeiros\source\repos\myclasses\myclassestest\bin\debug\net6.0\myclassestest.dll" priority="3" id="6f90c9b0-a18c-f05c-3cc2-7cca266a69be">
      <Owners>
        <Owner name="JhonatanM" />
      </Owners>
      <TestCategory>
        <TestCategoryItem TestCategory="NoException" />
      </TestCategory>
      <Execution id="3de833ab-8b6c-48b3-bcc0-1011e387307e" />
      <TestMethod codeBase="C:\Users\Jhonatan Medeiros\source\repos\MyClasses\MyClassesTest\bin\Debug\net6.0\MyClassesTest.dll" adapterTypeName="executor://mstestadapter/v2" className="MyClassesTest.FileProcessTest" name="FileNameDoesNotExist" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="928aa985-ca73-c448-5d8b-b6e6f1f3a744" executionId="d9504737-38ba-4273-8350-42c8c8d184d4" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="6f90c9b0-a18c-f05c-3cc2-7cca266a69be" executionId="3de833ab-8b6c-48b3-bcc0-1011e387307e" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="8b48ffd8-8c07-5f7b-0eb6-d4aee87d6161" executionId="91db61bc-1971-4e82-8c32-701ae4b1f092" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="0afa7d29-06f5-3a62-8ce9-439993ab8178" executionId="e7591804-9502-4017-99f9-73069e63f766" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="e40784d4-b509-7b24-dc06-ffcaafa43c7b" executionId="c0cb277d-555c-45e1-9da0-579173c4f4c4" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Resultados que não estão em uma lista" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="Todos os resultados carregados" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Completed">
    <Counters total="5" executed="4" passed="4" failed="0" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>O teste 'SimulateTimeout' foi ignorado na execução de teste.&#xD;
</StdOut>
    </Output>
  </ResultSummary>
</TestRun>